src/textForCLIAndError.ts

```diff
@@ -46,6 +47,9 @@ export enum ErrorMessageExt {
 }
 
 export enum textForInfoMsg{
+    downloadFlameExtManually = "You can manually download the extension 'Flame Chart Visualizer for JavaScript Profiles' at URL: https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-js-profile-flame (or) Goto  Extensions(shift+cmd+X) in vscode terminal --> search for 'Flame Chart Visualizer for JavaScript Profiles' --> click on install",
+    cpuprofileFileDownloadSuccess = 'Successfully downloaded the extension Flame Chart Visualizer for JavaScript Profiles.',
+    cpuProfileOptionMsg = `To view '.cpuprofile' files you should have extension 'Flame Chart Visualizer for JavaScript Profiles'. Click on 'Download' to automatically download the extension.`,
```

Comment:
```suggestion
    cpuProfileOptionMsg = `To view '.cpuprofile' files you should have extension 'Flame Chart Visualizer for JavaScript Profiles.' Click on 'Download' to automatically download the extension.`,
```

Reply:
I think this sugesstion is by mistake

Reply:
It's a minor thing but American style quoting puts punctuation inside quote marks even when they're not part of a quoted text, and American style text is the default for UI.

So either way, I'm not overly picky about this.

Reply:
hoo I did not knew this.. will change NP